src/test_storageiface.py

```diff
@@ -0,0 +1,139 @@
+'''
+test_storageiface.py
+
+Basic unit testing of the storage interfaces
+'''
+
+import unittest
+import logging
+import configparser
+
+class TestStorage(unittest.TestCase):
+  '''Simple tests for the storage layers of the WOPI server'''
+
+  def __init__(self, *args, **kwargs):
+    '''One-off initialization of the test environment: create mock logging and import the library'''
+    super(TestStorage, self).__init__(*args, **kwargs)
+    loghandler = logging.FileHandler('/tmp/wopiserver.log')
+    loghandler.setFormatter(logging.Formatter(fmt='%(asctime)s %(name)s[%(process)d] %(levelname)-8s %(message)s',
+                                              datefmt='%Y-%m-%dT%H:%M:%S'))
+    log = logging.getLogger('wopiserver.test')
+    log.addHandler(loghandler)
+    log.setLevel(logging.DEBUG)
+    # read the configuration
+    config = configparser.ConfigParser()
+    try:
+      with open('wopiserver-test.conf') as fdef:
```

Comment:
What does fdef stand for? (Maybe we can improve this naming?)

Reply:
yep, again effect of overlapping changes. A proper name is `fdconf` (`fd` being a common abbreviation for file descriptor)